src/gui/optionsdlg.cpp

```diff
@@ -95,7 +99,8 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
     m_ui->tabSelection->item(TAB_ADVANCED)->setIcon(GuiIconProvider::instance()->getIcon("preferences-other"));
     for (int i = 0; i < m_ui->tabSelection->count(); ++i) {
-        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 64));  // uniform size for all icons
+        // uniform size for all icons
+        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 62));
```

Comment:
OK.
The link doesn't work for me.
Merge the new commit into the **1st** one. And also rebase.
Then I can merge it.